tests/test_browser.py

```diff
@@ -2457,6 +2457,12 @@ def post_build(*args):
       open('test.html', 'w').write(html2)
     self.btest('sdl2_gl_frames_swap.c', reference='sdl2_gl_frames_swap.png', args=['--proxy-to-worker', '-s', 'GL_TESTING=1', '-s', 'USE_SDL=2'], manual_reference=True, post_build=post_build)
 
+  def test_sdl2_ttf(self):
+    shutil.copy2(path_from_root('tests/freetype','LiberationSansBold.ttf'), self.get_dir())
+    self.btest('sdl2_ttf.c', expected='1',
```

Comment:
instead of relying on report result, let's make this a reference test, see e.g. in the test before this, `self.btest('sdl2_gl_frames_swap.c', reference='sdl2_gl_frames_swap.png'`. The reference image is checked for a pixel perfect match.